.github/workflows/e2e.yml

```
@@ -31,6 +31,22 @@ jobs:
             echo "is_pr=false" >> "$GITHUB_OUTPUT"
           fi
 
+      - name: Add comment to PR
+        if: steps.check_pr.outputs.is_pr == 'true'
+        uses: actions/github-script@v5
```

```suggestion
        uses: actions/github-script@v5
```
We should probably harden this with a specific SHA

---

Is that really necessary for official actions from github? This is in line with how github documents usage. If github's own actions are compromised, that'd be pretty messed up.

@bjhargrave curious on your input here

---

Best practice for security it to use SHAs (and dependabot to keep things up-to-date). SHAs also important for reproducibility. v5 will move so if you make a branch from here in the future using a SHA will use the exact same revision of the action as is used now. So even if you fully trust GitHub, SHAs are still a best practice.

---

Thanks to you both for the input.

I updated this to use a hash, instead.